linode/manager
# Refactor: [M3-6522-copy-tooltip] CopyTooltip: Styled component and v7 story

## Description 📝
CopyTooltip small refactor to use styled components (already went through a MUI migration but it's cleaner that way) + removing old story and using a v7 one

No change in rendering, should look exactly the same. 

## Testing
- confirm no visual regression
- confirm story behavior at ?path=/docs/components-tooltip-copy-tooltip--documentation
